apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/disable-core-nux.js

```diff
@@ -22,5 +22,6 @@ subscribe( () => {
 	if ( select( 'core/edit-post' )?.isFeatureActive( 'welcomeGuide' ) ) {
 		dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' );
 		dispatch( 'automattic/nux' ).setWpcomNuxStatus( { isNuxEnabled: true } );
+		dispatch( 'automattic/nux' ).toggleWpcomTourManualOpenStatus( { isTourManuallyOpen: true } );
 	}
 } );
```

Comment:
My assessment of the above lines of code.  These are triggered when user selects "Welcome Guide" from MoreMenu
22) check if the welcomeGuide has been activated in core
23) toggle the feature in core (presumably turning it off) since it was just activated
24) set a new property that explicitly flags when the user manually opens the Tour 

We need a way to differentiate the Tour (or NUX modal) being opened automatically (the first time user opens editor) vs when the user opens it from the More Menu.

A better refactor would be to contain the usage of the `isNuxEnabled` purely for the first time viewing of the Tour (or NUX modal)  new-user-experience, thus respecting the '`nux`' meaning.   I'm not sure if it is wise to do that refactor now, or wait until after A/B testing.  My gut feeling is leave a comment and tidy up after A/B testing.